bin/train.py

```diff
@@ -306,7 +307,9 @@
     )
     callback_list.append(checkpoint_callback)
     callback_list.append(LearningRateMonitor(logging_interval="step"))
-callback_list.append(EarlyStopping(monitor="val_mean_loss", mode="min", patience=50))
+    callback_list.append(
+        EarlyStopping(monitor="val_mean_loss", mode="min", patience=50)
+    )
```

Comment:
I think you can choose no-log mode and still want to test the early stopping callback, so I would keep it as it was before.

Reply:
Unfortunately I tried and it doesn't work...early stopping needs the "val_mean_loss" to be logged so it wouldn't be a "no_log" mode anymore if we log stuff.